packages/connectors-lib/src/config.js

```diff
@@ -42,7 +42,8 @@ export default {
     scope: process.env.MSPA_SCOPE,
     client: {
       url: process.env.MSPA_CLIENT_URL,
-      timeout: process.env.MSPA_CLIENT_TIMEOUT_MS
+      timeout: process.env.MSPA_CLIENT_TIMEOUT_MS,
```

Comment:
You might want to set some defaults here. E.g. 

```
timeout: process.env.MSPA_CLIENT_TIMEOUT_MS || 60000,
```

Reply:
Oh scratch that. I can see that you're setting your defaults in the actual controller. That's fine.